Xamarin.Forms.Core/Label.cs

```diff
@@ -127,10 +164,43 @@ public LineBreakMode LineBreakMode
 			set { SetValue(LineBreakModeProperty, value); }
 		}
 
+		string ToTitleCase(string input)
+		{
+#if NETSTANDARD1_0
+			StringBuilder sb = new StringBuilder(input.Length);
+			var beginningOfWord = true;
+			var wordSeparatorChars = new[] { ' ', '\t', '\n', '.', ',', ';', '-', ':' };
+			for (var i = 0; i < input.Length; i++)
+			{
+				var c = input[i];
+				if (char.IsWhiteSpace(c) || wordSeparatorChars.Contains(c))
+				{
+					sb.Append(c);
+					beginningOfWord = true;
+					continue;
+				}
+				if (beginningOfWord)
+				{
+					sb.Append(char.ToUpperInvariant(c));
+					beginningOfWord = false;
+					continue;
+				}
+				sb.Append(char.ToLowerInvariant(c));
+			}
+			return sb.ToString();
+#else
+			return System.Globalization.CultureInfo.CurrentCulture.TextInfo.ToTitleCase(input.ToLower());
```

Comment:
TitleCase is more complex than this, some words shouldn't be capitalized.

Reply:
TitleCase is valid for english texts. Different culture capitalize their texts differently